src/lib/atoms/HelperText/HelperText.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { ReactWrapper, mount } from 'enzyme';
+
+// Components
+import HelperText, { IHelperTextProps } from './index';
+import GeneUIProvider from '../../providers/GeneUIProvider';
+import { Icon } from '../../../index';
+
+describe('HelperText ', () => {
+    let setup: ReactWrapper<IHelperTextProps>;
+    beforeEach(() => (setup = mount(<HelperText text={'test'} />, { wrappingComponent: GeneUIProvider })));
+
+    it('renders without crashing', () => {
+        expect(setup.exists()).toBeTruthy();
+    });
+
+    it.each<IHelperTextProps['size']>(['medium', 'small'])('should have "%s" size', (size) => {
+        const wrapper = setup.setProps({ size });
+
+        expect(wrapper.find('.helperText').hasClass(`helperText_size_${size}`)).toBeTruthy();
+    });
+
+    it.each<IHelperTextProps['type']>(['rest', 'danger', 'warning'])('should have "%s" type', (type) => {
+        const wrapper = setup.setProps({ type });
+
+        expect(wrapper.find('.helperText').hasClass(`helperText_type_${type}`)).toBeTruthy();
```

Comment:
` expect(wrapper.find(`.helperText_type_${type}`).exist()).toBeTruthy();`

Reply:
We can't change like this because the current solution fits the logic of the problem better.
Your proposed solution may cause a problem in the context of the scope.